clients/ts/signalr/src/IHttpClientOptions.ts

```diff
@@ -0,0 +1,5 @@
+/** Options provided to {@link @aspnet/signalr.DefaultHttpClient} and {@link @aspnet/signalr.XhrHttpClient} to configure options for the HTTP-based transports. */
+export interface IHttpClientOptions {
+    /** Disable withCredentials for the default {@link @aspnet/signalr.HttpClient} */
+    disableXhrWithCredentials?: boolean,
```

Comment:
Let's just call this `withCredentials` and treat undefined as `true`. Negative booleans are weird :)